CMakeLists_files.cmake

```diff
@@ -93,15 +97,16 @@ list (APPEND TEST_SOURCE_FILES
   tests/test_minpvprocessor.cpp
 #	tests/grid_test.cc
   tests/p2pcommunicator_test.cc
-  tests/test_regionmapping.cpp
   tests/test_repairzcorn.cpp
   tests/test_sparsetable.cpp
-  tests/test_ug.cpp
   tests/test_quadratures.cpp
 	)
 
-if(opm-parser_FOUND)
-  list(APPEND TEST_SOURCE_FILES tests/test_compressedpropertyaccess.cpp)
+if(HAVE_ECL_INPUT)
+  list(APPEND TEST_SOURCE_FILES
+		tests/test_regionmapping.cpp
+    tests/test_ug.cpp
```

Comment:
meh.

Reply:
I agree. That said, it seems like these tests use stuff from opm-parser and this is already used in this file..

Reply:
i was only referring to the indent.

Reply:
ah, I see. the reason is that the modeline at the beginning of the file instructs emacs to use tabs instead of spaces and the tab with should be assumed to be 2 while github uses a tabwidth of 8. (i.e., in my emacs the indentation is fine).